docs/installation-guide-aws.md

```diff
@@ -0,0 +1,147 @@
+# Installation steps
+
+## Prerequisites
+
+Common:
+- `aws` cli
+
+### Create S3 bucket and DynamoDB table for terraform state
+
+```
+export CLUSTER="cluster1"
```

Comment:
IN few lines below there is used 'gauss' instead of 'cluster1'  I would pick one and stay consistent